### PR TITLE
Create percentage bars in Stats cards

### LINKED ISSUE #9459: Update row type to include percentage in Stats
Add a bar that contains the percentage of the top value in the list items in Stats blocks in posts / pages, tags / categories, authors (the highest value would be 100% - a bar that stretches under the whole textview). The color of the bar is `#c8d7e1`, height is `4dp`, the designs are attached:

Screenshot 2019-03-28 at 08 55 19

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #9459
This PR adds the percentage bar to the Posts/Pages, Authors and Tags/Categories. The logic is that the biggest value (by views) gets 100% width and the other values get width by the percentage of the biggest value. I've created a drawable for the rounded corners of the progress bar that's used for showing the progress. I've also noticed the `subtext` field was never used so I removed it. 

To test:
* Go to Stats/DWMY
* Check that the Posts/Pages block contains the percentage bar
* Check that the lines with the bar are higher
* Check that the lines without the bar are still the same height
* Check that the Authors block contains the percentage bar
* Expand an Author
* Check that the author's posts don't have the percentage bar
* Check that the author's posts have lower height than authors
* Go to Insights/Tags&Categories
* Check that the items have the percentage bar
* Check that the expanded category items don't have the percentage bar

Screenshot_1561012756
Screenshot_1561012748

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by SylvesterWilmott** [comment]
Looks fantastic @planarvoid , thanks!

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/AuthorsUseCase.kt [hunk 1]
```diff
@@ -148,6 +150,17 @@ constructor(
         return items
     }
 
+    private fun getBarWidth(
```
Comment (khaykov) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/AuthorsUseCase.kt]: Do you think we can extract this function to some Utils class, as we are using it all around the place?

**Review by khaykov — COMMENTED** [review]
Bar looks fancy and works well ✨ I left just one small comment about the code :)

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/AuthorsUseCase.kt (hunk 1 above)
Comment (planarvoid) [reply WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/AuthorsUseCase.kt]: I was thinking about it :-), I think it makes sense

**Comment by planarvoid** [comment]
@khaykov thanks for the review, I've extracted the common utility method and wrote some tests 👍

**Review by khaykov — APPROVED** [review]
Thanks for the fixes! Looks great 👍

**Comment by designsimply** [comment]
Beta tested with alpha-177 and it looks great! 👍

Adding a couple screenshots for reference:

Screenshot_20190703-115236 Screenshot_20190703-115303  Screenshot_20190703-115521

**Pull request merged**